### PR TITLE
ILS/LOC fixes

### PR DESCRIPTION
**Pull request by LawinS**
LTBZ added (#2546)
LTCV added (#2570)
LTCM added (#2580)
LTCP added (#2581)
LTCS added (#2582)

### CODE REVIEW

File: Glideslope.json [hunk 1]
```diff
@@ -31161,6 +31187,32 @@
...
+   "name": "GS",
+   "airportICAO": "LTCM",
+   "identifier": "ISDP",
+   "associatedRunwayNumber": "23",
+   "frequency": 108900,
+   "elevation": 32,
+   "latitude": 42.02378830,
+   "longitude": 35.08172365,
+   "bearing": 233.54,
+   "glideslope": 3.00023354,
```
Comment (velocity23) [inline Glideslope.json:31200]: Should be 3.0023354, as your bearing is 233.54 not 23.354

File: Glideslope.json [hunk 2]
```diff
@@ -31161,6 +31187,32 @@
...
+   "name": "GS",
+   "airportICAO": "LTCP",
+   "identifier": "IADY",
+   "associatedRunwayNumber": "04",
+   "frequency": 110500,
+   "elevation": 2214,
+   "latitude": 37.72397095,
+   "longitude": 38.45827215,
+   "bearing": 49.23,
+   "glideslope": 3.00004923,
```
Comment (velocity23) [inline Glideslope.json:31213]: Should be 3.0004923 as your bearing is 49.23 not 4.923

File: Glideslope.json [hunk 3]
```diff
@@ -31187,6 +31252,19 @@
...
+  "name": "GS",
+   "airportICAO": "LTCV",
+   "identifier": "ISNK",
+   "associatedRunwayNumber": "11",
+   "frequency": 108350,
+   "elevation": 2041,
+   "latitude": 37.37054369,
+   "longitude": 42.04292567,
+   "bearing": 115.68,
+   "glideslope": 3.00011568,
```
Comment (velocity23) [inline Glideslope.json:31265]: Should be 3.0011568 as your bearing is 115.68 not 11.568

**Review by velocity23 — CHANGES_REQUESTED** [review]

**Comment by LawinS** [comment]
Hopefully, it should be good now.

**Review by velocity23 — APPROVED** [review]

**Pull request merged**